CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ Todos los cambios notables a este proyecto serán docuemntados en este archivo.
 El formato está basado en [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 y este proyecto adhiere a [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.1.0] - 2019-04-04
+### Added
+- Se agregaron los parámetros `qr_width_height` y `commerce_logo_url` a Options. Ahora puedes configurar estos parámetros globalmente o por transacción.
```

Comment:
@ffiebig Remember that we must treat the developers as final users, and you're not explaining what allows that new feature. I suggest something like:

- Se agregaron los parámetros `qr_width_height` y `commerce_logo_url` a Options, para especificar el tamaño del QR generado para la transacción, y especificar la ubicación del logo de comercio para ser mostrado en la aplicación móvil de Onepay. Puedes configurar estos parámetros globalmente o por transacción.

And is mandatory to be more specific explaining this, because this is the first version of the SDK which includes both parameters.

Reply:
Changes made